src/main/java/com/renderLatex/service/RenderService.java

```diff
@@ -1,90 +1,139 @@
 package com.renderLatex.service;
 
 import com.renderLatex.entities.LatexContent;
+import org.apache.batik.dom.GenericDOMImplementation;
+import org.apache.batik.svggen.SVGGeneratorContext;
+import org.apache.batik.svggen.SVGGraphics2D;
+import org.apache.commons.io.FileUtils;
+import org.apache.pdfbox.Loader;
+import org.apache.pdfbox.printing.PDFPageable;
+import org.apache.pdfbox.rendering.ImageType;
+import org.apache.pdfbox.rendering.PDFRenderer;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.w3c.dom.DOMImplementation;
+import org.w3c.dom.Document;
 
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
 import java.io.*;
+import javax.imageio.*;
+import java.awt.image.BufferedImage;
 import java.nio.file.Paths;
+import java.util.Locale;
+import java.util.UUID;
 
 @Service
 public class RenderService {
     private String latexDocClass = "\\documentclass[border=0.50001bp,convert={convertexe={imgconvert},outext=.png}]{standalone} \n";
     private final String docStart = "\\begin{document} \n";
     private final String docEnd = " \\end{document} \n";
-    private String packages = "";
-    private String content = "";
+    private final String userDirectory = Paths.get("").toAbsolutePath().toString();
+    private final String tempDirectory = Paths.get("").toAbsolutePath().toString()+"/tmp";
+
+
 
     @Autowired
     public RenderService(){
+        File file = new File(userDirectory +"/tmp");
+        try{
+            FileUtils.deleteDirectory(file);
+            if (file.mkdir()){
+                System.out.println("directory created");
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
 
     }
 
-    public void renderAsPdf(LatexContent latexContent){
-        this.content = latexContent.getContent();
-        this.packages = "";
-        latexContent.getLatexPackages().forEach(item -> this.packages += (item + " \n "));
-        createTexDoc();
-        renderTex("renderFile.tex");
+//    private string prepareEnvi
+
+    public String renderAsPdf(LatexContent latexContent){
+        final String uuid = UUID.randomUUID().toString();
+        final String currentImagePath = tempDirectory + "/" + uuid;
```

Comment:
Do not use string concatenation to build paths if possible.
Also ideally refactor path creation and folder creation into one helper method.